app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -228,11 +228,13 @@ class EventDetailsFragment : Fragment() {
         }
 
         // Set Cover Image
-        event.originalImageUrl?.let {
-            Picasso.get()
+        event.originalImageUrl.let {
+            if (it != null) {
```

Comment:
Removing this additional check is the entire point of using ?.let

Reply:
Re-adding

Reply:
Resolved